### ISSUE TITLE
Confused on how to use custom rules

### ISSUE THREAD

**Issue by dondod** [JLLeitschuh/ktlint-gradle#695]
Hi, I have a gradle project with a main project (I'll call `main-project`) and another project that contains custom ktlint rules called `ktlint-custom-rules`.  I've verified that my rules work when run with the latest (0.50.0) command-line `ktlint` runner.  However, I haven't been able to get the custom rules to be recognized when running `ktlintCheck` against the `main-project`.

My `build.gradle` file in the `ktlint-custom-rules` looks like this:

```groovy
apply plugin: "kotlin"

dependencies {
    runtimeOnly("com.pinterest.ktlint:ktlint-rule-engine:0.50.0")
    runtimeOnly("com.pinterest.ktlint:ktlint-rule-engine-core:0.50.0")
    implementation("com.pinterest.ktlint:ktlint-cli-ruleset-core:0.50.0")
    testImplementation("com.pinterest.ktlint:ktlint-test:0.50.0")
}
```

In the `build.gradle` file in `main-project`, I have this:

```groovy
//...

plugins {
    id "org.jlleitschuh.gradle.ktlint" version "11.5.0"
}

apply plugin: 'org.jlleitschuh.gradle.ktlint'

dependencies {
    ktlintRuleset(project(":ktlint-custom-rules"))
}

```

Any ideas?  I'm guessing I'm missing something simple here.

**Comment by SumeraMartin** [comment]
Do you have META-INF/services/com.pinterest.ktlint.cli.ruleset.core.api.RuleSetProviderV3 file created? (It was required in the previous versions. Currently it is not working for me and I have a feeling that it is broken in this library.)

 https://github.com/pinterest/ktlint/blob/master/ktlint-cli-ruleset-core/src/main/kotlin/com/pinterest/ktlint/cli/ruleset/core/api/RuleSetProviderV3.kt

**Comment by wakingrufus** [comment]
Looks like my implementations for ktlint 0.49 and 0.50 don't properly load custom rulesets from the classpath. This was reported in #697 as well. I will work on a fix

**Comment by wakingrufus** [comment]
This should be fixed in 11.5.1. Please try again
